pkg/webhook/webhook.go

```diff
@@ -217,3 +265,16 @@ func HandleHooks(ctx context.Context, rContext *types.Context) http.Handler {
 		root)
 	return root
 }
+
+// git ref docs: https://git-scm.com/book/en/v2/Git-Internals-Git-References
+func getBranchTagFromRef(ref string) (string, string) {
+	if strings.HasPrefix(ref, branchRefPrefix) {
+		return strings.TrimPrefix(ref, branchRefPrefix), ""
+	}
+
+	if strings.HasPrefix(ref, tagRefPrefix) {
+		return "", strings.TrimPrefix(ref, tagRefPrefix)
+	}
+
+	return "", ""
+}
```

Comment:
We should have a unit test for this.

Reply:
Yeah it's tiny, but I was just on a call about unit tests and thought this could be a footgun. Maybe trying multiple payloads for this could be nice too!

Reply:
yes, testing payload is actually a good idea. Should be adding that later